provisioning/playbook.yml

```diff
@@ -26,10 +28,11 @@
     - { role: geerlingguy.apache, when: drupalvm_webserver == 'apache' }
     - { role: geerlingguy.apache-php-fpm, when: drupalvm_webserver == 'apache' }
     - { role: geerlingguy.nginx, when: drupalvm_webserver == 'nginx' }
-    - geerlingguy.mysql
     - geerlingguy.php
     - geerlingguy.php-pecl
-    - geerlingguy.php-mysql
+    - { role: geerlingguy.mysql, when: drupalvm_database == 'mysql' }
+    - { role: geerlingguy.php-mysql, when: drupalvm_database == 'mysql' }
+    - { role: ANXS.postgresql, when: drupalvm_database == 'pgsql' }
```

Comment:
I don't know if this makes it simpler or not... but we could also allow installation of _both_ pg and mysql. Though I can't think of many people who would ever need it.